conf/logback.xml

```diff
@@ -14,7 +14,10 @@
   <logger name="org.sagebionetworks.bridge" level="DEBUG" additivity="false">
     <appender-ref ref="STDOUT" />
   </logger>
-  <logger name="org.springframework" level="INFO" additivity="false">
+  <logger name="org.sagebionetworks.bridge.play" level="ERROR" additivity="false">
```

Comment:
Our play controllers should have error level of INFO or lower. If we log it, we want to see it. If we don't want to see it, we shouldn't log it.

Specifically, this means we lose things like MetricsInterceptor.